docs/Reference/architecture/CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# CONTRIBUTING
+
+## Installation
+
+Cosmos Cash diagrams are based on PlantUML. 
+
+PlantUML can be installed on your local system through your platforms' package manager:
+
+MacOS, through [Homebrew](https://brew.sh/)
+
+```
+> brew install plantuml
+```
+
+Windows, through [Chocolatey](https://chocolatey.org/)
+
+```
+> choco install plantuml
+```
+
+Linux-based systems, through your distributions package manager
+
+## Contributing diagrams
```

Comment:
let's use our standard Title case style capitalization for all headings, I did not not all of them below, it's handy to use https://capitalizemytitle.com/